src/main/scala/uk/gov/dhsc/htbhf/ClaimSimulation.scala

```diff
@@ -125,14 +140,14 @@ class ClaimSimulation extends Simulation {
     .exec(http("send_code")
       .post("/send-code")
       .formParam("channelForCode", "text")
-      .formParam("_csrf", "${csrf_token}")
+      .formParam("_csrf", "${csrf_token2}")
```

Comment:
Not specific to this PR, but the send-code and enter-code performance tests were failing with the wrong csrf_token, although slightly worryingly, this wasn't causing the performance tests to fail.